### PR TITLE
feat: middleware upgrade to work with chain breaking changes

### PR DESCRIPTION
**Pull request by VictorVicente**
Claim scopes have changed to an object in the form of {type; value}

BREAKING CHANGE: Scope filtering in `getIdentitiesWithClaims` and `getTargetingClaims` take an
`Scope` object instead of a string.

### CODE REVIEW

File: src/Claims.ts [hunk 1]
```diff
@@ -124,7 +132,7 @@ export class Claims {
     const result = await context.queryMiddleware<Ensured<Query, 'didsWithClaims'>>(
       didsWithClaims({
         dids: targets?.map(target => signerToString(target)),
-        scope,
+        scope: scope ? { type: ClaimScopeTypeEnum[scope.type], value: scope.value } : scope,
```
Comment (monitz87) [inline src/Claims.ts:135]:
```suggestion
        scope: scope ? { type: ClaimScopeTypeEnum[scope.type], value: scope.value } : undefined,
```

Just to be more explicit

File: src/Claims.ts [hunk 2]
```diff
@@ -251,7 +259,7 @@ export class Claims {
     const result = await context.queryMiddleware<Ensured<Query, 'issuerDidsWithClaimsByTarget'>>(
       issuerDidsWithClaimsByTarget({
         target: did,
-        scope,
+        scope: scope ? { type: ClaimScopeTypeEnum[scope.type], value: scope.value } : scope,
```
Comment (monitz87) [inline src/Claims.ts:262]:
```suggestion
        scope: scope ? { type: ClaimScopeTypeEnum[scope.type], value: scope.value } : undefined,
```

File: src/utils/__tests__/index.ts [hunk 3]
```diff
@@ -1962,7 +1962,7 @@ describe('createClaim', () => {
   test('', () => {
```
Comment (monitz87) [inline src/utils/__tests__/index.ts:1962]: Can you add a description for this test?

**Review by monitz87 — CHANGES_REQUESTED** [review]

File: src/utils/__tests__/index.ts (hunk 3 above)
Comment (VictorVicente) [reply src/utils/__tests__/index.ts]: Done

**Review by monitz87 — APPROVED** [review]

**Pull request merged**